sql/driver/driver_test.go

```diff
@@ -48,9 +49,10 @@ func TestPlaceholders(t *testing.T) {
 	s, db := setup(t)
 	defer cleanup(s, db)
 
-	year, month, day := 3015, time.August, 30
-	timeVal := time.Date(year, month, day, 3, 34, 45, 345670000, time.UTC)
-	dateVal := time.Date(year, month, day, 0, 0, 0, 0, time.UTC)
+	timeVal := time.Date(3015, time.August, 30, 3, 34, 45, 345670000, time.UTC)
+	// date must be in local time because the server will return a date
+	// in local time.
```

Comment:
I'm confused about the need to have date as local, but the timeVal above as UTC?